aspnet/fundamentals/configuration.rst

```diff
@@ -41,7 +40,6 @@ It's not unusual to store configuration values in a hierarchical structure, espe
 .. _config-json:
 
 .. literalinclude:: /../common/samples/WebApplication1/src/WebApplication1/appsettings.json
-  :linenos:
   :language: json
 
 The application uses configuration to configure the right connection string. Access to the ``ConnectionString`` setting is achieved through this key: ``Data:DefaultConnection:ConnectionString``.
```

Comment:
Ah, ok. The common samples have now been updated. Can you rebase and force push your branch then so we can see what the real final doc will look like?